automated_provisioning/README.md

```diff
@@ -75,6 +75,29 @@ The command should look something like the below (with the parameters replaced b
 ./ps/ConnectSiloToOrchestrator.ps1 -SubscriptionId_Orchestrator "Your-Orchestrator-SubscriptionId" -AMLWorkspaceName "Your-Orchestrator-Workspace-Name" -AMLWorkspaceRGName "Your-Orchestrator-Resource-Group-Name" -AMLWorkspaceLocation "Your-Orchestrator-Location" -K8sClusterName "Name-of-K8s-Cluster-to-Connect" -AMLComputeName "AML-Compute-Name-to-Create"
 ```
 
+### Change compute intense size
+The default compute instance only provision a small portion of your k8s cluster (specifically, 1.5G memory and 0.6 cpu). You may need to override this by the following steps.
+
+Setup your local kubectl environment and connect to the aks. In the PowerShell, run the following
+```
+az login
+az account set --subscription "subscription-of-Cluster"
```

Comment:
Rather than adding manual docs, can you add a PowerShell script which applies this logic?